src/ls/clients/regal.ts

```diff
@@ -143,6 +157,9 @@ export function activateRegal(_context: ExtensionContext) {
         outputChannel: outChan,
         traceOutputChannel: outChan,
         revealOutputChannelOn: 0,
+        connectionOptions: {
+            messageStrategy: new debuggableMessageStrategy,
```

Comment:
I was mainly keen to have _something_ here so that we had it as bookmark so to speak. As it took my some time work out that this was an option and did what I thought it did.